hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -88,9 +87,15 @@
    */
   public static final String FALLBACK_GROUPS_KEY = "hbase.rsgroup.fallback.groups";
```

Comment:
I thought this is not released now. For Availability, how about allow any table fallback to any other rsgroups? And only introduced one config.

Reply:
OK, I'll refactor this feature. For all tables, whether or not system tables, have same logic: priority fallback to the default group, and then fallback to any group.